reo/validators.py

```diff
@@ -1570,6 +1615,9 @@ def validate_urdb_response(self, number=1):
                     self.urdb_errors += rate_checker.errors
             except:
                 self.urdb_errors.append('Error parsing urdb rate in %s ' % (["Scenario", "Site", "ElectricTariff"]))
+        else:
+            self.urdb_errors.append('Invalid URDB response: %s'.format(str(urdb_response)))
```

Comment:
Should this be "Invalid type for `urdb_response`: got {} but must be a `dict`." ?

Reply:
It's possible the user specified urdb_label or urdb_utility_name/urdb_rate_name to get the response so I thought it would be better to keep it more generic and just show what the non-compliant content is